### DISCUSSION TITLE
Beatmap skins should not change skin layout

### DISCUSSION THREAD

**Discussion by respektive** [ppy/osu#27201] [Q&A] [answered]
I like to play old maps with the beatmap skins enabled, cause often times they contain some cool assets. The issue I have with it however is, that I have edited my skin's HUD to fit my linking and it just replaces everything with the default skin. For example in my skin I have replaced the default key counter with the argon one and changed the position of it.

I think the beatmap skin option should just use the sprites from the skin and not change any of the HUD. At least as long as the beatmap skin doesn't contain any of the new skinning options in lazer. I can see maps using those in the future to create some cool experiences, but I am not aware of any at this point in time.

Example map that's using a custom skin: https://osu.ppy.sh/beatmapsets/12448#osu/47356
My skin that has some edits to the HUD: https://drive.google.com/file/d/1GXp3NzJWd5INygzcmYANZ8-H4Q5cdDIG/view?usp=drive_link

**Comment by Rekunan** [comment]
Maybe could also go further and add an option for specific elements that you'd like to stay the same even after enabling beatmap skins, in particular keeping 300's as invisible when a lot of older beatmap skins add them.

**Comment by respektive** [comment, marked as answer]
This is actually resolved by https://github.com/ppy/osu/pull/28739
